vdoctl/cmd/root.go

```diff
@@ -156,3 +158,23 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
 	return nil
 }
+
+func checkVdoDeployment() {
```

Comment:
lets pass the context from caller rather than creating a new one in this function

Reply:
can we rename the function to getVDONamespace